buster/completers/chatgpt.py

```diff
@@ -28,6 +28,12 @@
 
 class ChatGPTCompleter(Completer):
     def __init__(self, completion_kwargs: dict, client_kwargs: Optional[dict] = None):
+        """Initialize the ChatGPTCompleter with completion and client keyword arguments.
```

Comment:
I would have docstring for the class, rather than for the init, to be consistent across the project